scss/_variables-dark.scss

```diff
@@ -53,6 +53,8 @@ $headings-color-dark:               $white !default; // Boosted mod: instead of
 $link-color-dark:                   $white !default; // Boosted mod: instead of `tint-color($primary, 40%)`
 $link-hover-color-dark:             $primary !default; // Boosted mod: instead of `shift-color($link-color-dark, -$link-shade-percentage)`
 $code-color-dark:                   tint-color($code-color, 40%) !default;
+$mark-color-dark:                   $body-color-dark !default;
+$mark-bg-dark:                      $yellow-800 !default;
```

Comment:
I'm fine with the regression on dark mode since it will be changed by our implementation.
FYI, on main you need to set a `.bg-dark` somewhere to make the magic happen atm.